src/components/scenes/SwapSettingsScene.tsx

```diff
@@ -307,8 +309,8 @@ export const SwapSettingsScene = connect<StateProps, DispatchProps, OwnProps>(
     changePreferredSwapPluginType(swapPluginType) {
       dispatch(setPreferredSwapPluginType(swapPluginType))
     },
-    ignoreAccountSwap() {
-      dispatch(ignoreAccountSwap())
+    async ignoreAccountSwap() {
+      await dispatch(ignoreAccountSwap())
```

Comment:
Awesome. It's always a good sign when we do `await dispatch`. It means we have correctly piped the promise up to the top of the call stack. 👍